Integrations/integration-Pipl.yml

```diff
@@ -0,0 +1,280 @@
+            requestUrl,
+            {
+                Method: 'POST',
+                Headers: {
+                    'Content-Type': ['application/x-www-form-urlencoded']
+                },
+                Body: 'key=' + params.key
+            },
+            params.insecure,
+            params.proxy
+        );
+
+        if (res.StatusCode < 200 || res.StatusCode >= 300) {
+            throw 'Request Failed.\nStatus code: ' + res.StatusCode + '.\nBody: ' + JSON.stringify(res) + '.';
+        }
+        return JSON.parse(res.Body);
+    };
+
+    var createEntry = function(response) {
+        var data = [];
+        ec = {};
+        ec.Account =[];
+        //Only one person
+        if (response.person) {
+            data[0] = addPerson(response.person);
+            ec.Account[0] = buildEC(data, 0);
+            data[0]['Emails'] = '';
+            for (var j=0; j<data[0].Email.length; j++) {
+                data[0]['Emails'] += data[0].Email[j].Address + '\n';
+            }
+            delete data[0].Email;
+        } else {
+        //More than one person
+            for (var i=0; i<response.possible_persons.length; i++) {
+                data[i] = addPerson(response.possible_persons[i]);
+                ec.Account[i] = buildEC(data, i);
+                data[i]['Emails'] = '';
+                for (var j=0; j<data[i].Email.length; j++) {
+                    data[i]['Emails'] += data[i].Email[j].Address + '\n';
+                }
+                delete data[i].Email;
+            }
+        }
+
+        return {
+                    Type: entryTypes.note,
+                    ContentsFormat: formats.table,
+                    Contents: data,
+                    ReadableContentsFormat: formats.table,
+                    //HumanReadable: tableToMarkdown('Pipl', data, columns),
+                    //HumanReadable: data,
+                    EntryContext: ec
+                };
+    };
+
+    var buildEC = function(data, i) {
+        return {
+                    Addresses : data[i].Addresses,
+                    Email : data[i].Email,
+                    IDs : data[i].UserIDs,
+                    Names: data[i].Names,
+                    Phones: data[i].Phones,
+                    Usernames: data[i].Usernames
+                };
+    };
+
+    var addPerson = function(person) {
+        var response = {
+            Names: '',
+            Phones: '',
+            Gender: '',
+            DoB: '',
+            Image: '',
+            Usernames: '',
+            Email: '',
+            Educations: '',
+            UserIDs: '',
+            URLs: '',
+            Jobs: '',
+            Addresses: ''
+        };
+        if (person.names) {
+            for (var i=0; i<person.names.length; i++) {
+                response['Names'] += person.names[i].middle ? person.names[i].first + ' ' + person.names[i].middle + ' ' + person.names[i].last + '\n' : person.names[i].first + ' ' + person.names[i].last + '\n';
```

Comment:
`response['Names']` will give you lint warning. In JS we should use `response.Names`.